src/tests/WebApi.OpenAPI.SystemTest/WebApi.OpenAPI.SystemTest.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I don't think you need the "system" tests project. I think you can use a classic xunit test.
- Use the `Microsoft.AspNetCore.Mvc.Testing` package to start the server and call `/swagger/openapi-v1.json` (not the right URL, but should be similar)
- Validate the content using a tool such as Verify or Meziantou.Framework.InlineSnapshotTesting.

Reply:
I think this could be a good idea! I think that could be a good improvements in a separate PR